src/main/java/org/cryptomator/ui/error/ErrorController.java

```diff
@@ -100,8 +131,75 @@ public void copyDetails() {
 		});
 	}
 
-	/* Getter/Setter */
+	private void loadHttpResponse(HttpResponse<InputStream> response){
+		if (response.statusCode() == 200) {
+			Map<String,ErrorDiscussion> map = new Gson().fromJson(
+					new InputStreamReader(response.body(),StandardCharsets.UTF_8),
+					new TypeToken<Map<String,ErrorDiscussion>>(){}.getType());
+
+			if(!map.values().stream().filter(this::isPartialMatchFilter).findFirst().isEmpty()) {
+				Comparator<ErrorDiscussion> comp = this::compareExactMatch;
+				matchingErrorDiscussion = map.values().stream().sorted(comp
+						.thenComparing(this::compareSecondLevelMatch)
+						.thenComparing(this::compareThirdLevelMatch)
+						.thenComparing(this::compareIsAnswered)
+						.thenComparing(this::compareUpvoteCount)
+				).findFirst().get();
+				lookUpSolutionVisibility.set(true);
+			}
+		}
+	}
+
+	public boolean isPartialMatchFilter(ErrorDiscussion errorDiscussion) {
+		return errorDiscussion.title.contains(" " + errorCode.methodCode());
+	}
+
+	public int compareUpvoteCount(ErrorDiscussion ed1, ErrorDiscussion ed2) {
+		return Integer.compare(ed2.upvoteCount, ed1.upvoteCount);
+	}
```

Comment:
Can you add documation, on what condition of the upvote counts the function returns what?